src/Tickets/Emails/Email/Ticket.php

```diff
@@ -55,15 +55,6 @@ class Ticket extends \TEC\Tickets\Emails\Email_Abstract {
 	 */
 	public $version = '1.0.0';
```

Comment:
The version will be just for the post type, so that we can perform updates if we need to (at any given point)